### PR TITLE
chore: Add simple example of 2d arrays

### PR DESCRIPTION
**Pull request by kevaundray**
# Description

This adds a simple example of using an array in an array

## Problem\*

Resolves 

## Documentation

- [ ] This PR requires documentation updates when merged.

  

  - [ ] I will submit a noir-lang/docs PR.

  

  - [ ] I will request for and support Dev Rel's help in documenting this PR.

  
  

## Additional Context

# PR Checklist\*

- [ ] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

**Comment by kevaundray** [comment]
Hmm seems I accidentally rebased ontop of #2793

File: tooling/nargo_cli/tests/execution_success/simple_2d_array/Nargo.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[package]
+name = "1_mul"
```
Comment (TomAFrench) [inline tooling/nargo_cli/tests/execution_success/simple_2d_array/Nargo.toml:2]:
```suggestion
name = "simple_2d_array"
```

Not having this match breaks the `rebuild` script.

**Review by TomAFrench — APPROVED** [review]
Are we wanting to add support for this in the ABI?

**Comment by kevaundray** [comment]
> Are we wanting to add support for this in the ABI?

Yep, I'll open up an issue for it

**Pull request merged**